### DISCUSSION TITLE
Multiple outputs for StationaryModel

### DISCUSSION THREAD

**Discussion by pdiercks** [pymor/pymor#2355] [Q&A] [answered]
Hi,
I was wondering how to implement multiple outputs, because as argument `output_functional` of `StationaryModel` only a single `Operator` is allowed. Say, we have an operator `O1: R^N -> R^1` and an operator `O2: R^N -> R^1`, how would we define the operator `O: R^N -> R^2`, where `O.apply(U, mu)` returns `[O1.apply(U, mu), O2.apply(U, mu)]`?

I guess `BlockOperator` & friends could be used in this case, i.e. using `BlockColumnOperator` since I would like to have a blocked range, but the source space is not blocked. However, I was not sure if this was the intended use case or if I am missing another operator (maybe in the `constructions` module) that might be more natural to use.

Best,
Philipp

**Comment by sdrave** [comment, marked as answer]
Hi @pdiercks, you are right: `output_functional=BlockColumnOperator([O1, O2])` should do exactly what you want.
